### PR TITLE
Remove static variable for JmsPluginConfig class in JmsSource

### PR DESCRIPTION
**Pull request by henry-cask**
Need to remove DEFAULT_MESSAGE_RECEIVE static variable in JmsPluginConfig since it is showing up to UI via reflection.

### CODE REVIEW

File: cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/templates/etl/realtime/sources/JmsSource.java [hunk 1]
```diff
@@ -279,25 +279,20 @@ public JmsProvider getJmsProvider() {
...
     @Name(JMS_DESTINATION_NAME)
     @Description("Name of the destination to get messages")
-    private final String destinationName;
+    private String destinationName;
 
     @Name(JMS_MESSAGES_TO_RECEIVE)
     @Description("Max number messages should be retrieved per poll. The default value is 50.")
     @Nullable
-    private final Integer messagesToReceive;
+    private Integer messagesToReceive = 50;
```
Comment (gokulavasan) [inline cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/templates/etl/realtime/sources/JmsSource.java]: I think you need to have a default constructor that assigns this value to messagesToReceive.

**Comment by henry-cask** [comment]
Successful build: http://builds.cask.co/browse/CDAP-RT17-1

File: cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/templates/etl/realtime/sources/JmsSource.java (hunk 1 above)
Comment (henry-cask) [inline cdap-app-templates/cdap-etl/cdap-etl-lib/src/main/java/co/cask/cdap/templates/etl/realtime/sources/JmsSource.java]: Ah ok, we need to document this behavior

**Comment by henry-cask** [comment]
@gokulavasan Updated the patch with default constructor.

**Comment by gokulavasan** [comment]
LGTM :+1:

**Comment by henry-cask** [comment]
Thanks @gokulavasan, merging it now.

**Pull request merged**